### PR TITLE
fix(dgw): fix streaming with RDM initiated sessions

### PR DESCRIPTION
**Pull request by irvingoujAtDevolution**
This one is waiting for #1136 and https://github.com/austinleroy/ebml-iterable/pull/12 to be merged first. It's not ready, don't review it yet.

Edit:
It's ready

### CODE REVIEW

File: crates/streamer/Cargo.toml [hunk 1]
```diff
@@ -21,6 +21,7 @@ webm-iterable = { version = "0.6", features = ["futures"] }
 cadeau = { version = "0.5", features = ["dlopen"] }
 thiserror = "2"
 num_cpus = "1.16.0"
+ebml-iterable = "0.6.3"
```
Comment (CBenoit) [inline crates/streamer/Cargo.toml:24]:
```suggestion
ebml-iterable = "0.6"
```

File: crates/streamer/src/streamer/iter.rs [hunk 2]
```diff
@@ -3,7 +3,7 @@ use std::io::Seek;
 use anyhow::Context;
 use cadeau::xmf::vpx::is_key_frame;
 use thiserror::Error;
-use tracing::debug;
+use tracing::{debug, trace};
```
Comment (CBenoit) [inline crates/streamer/src/streamer/iter.rs:6]: Did you use `#[macro_use]` above `extern crate tracing` in lib.rs?

File: crates/streamer/src/lib.rs [hunk 3]
```diff
@@ -7,4 +7,5 @@ pub use config::StreamingConfig;
 pub use streamer::reopenable_file::ReOpenableFile;
 pub use streamer::signal_writer::SignalWriter;
 pub use streamer::{webm_stream, Signal};
-use webm_iterable as _;
+// We need ebml_iterable for TagIterator::emit_master_end_when_eof method
+use ebml_iterable as _;
```
Comment (CBenoit) [inline crates/streamer/src/lib.rs:11]: I don't see anything imported from this crate. Am I missing something?

Comment (irvingoujAtDevolution) [reply crates/streamer/src/lib.rs:11]: we depends on webm_iterable which depends on embm_iterable 0.6.2 that does not have emit_master_end_when_eof method. So we manually include latest embm_iterable for that

File: crates/streamer/src/streamer/iter.rs (hunk 2 above)
Comment (irvingoujAtDevolution) [reply crates/streamer/src/streamer/iter.rs:6]: I can do this in separate PR

File: crates/streamer/src/lib.rs (hunk 3 above)
Comment (CBenoit) [reply crates/streamer/src/lib.rs:11]: You can update the Cargo.lock with `cargo update -p ebml_iterable` instead. It’s fine to remove the dependency from our Cargo.toml.

**Review by CBenoit — APPROVED** [review]
LGTM!

**Pull request merged**